cockatrice/src/messagelogwidget.cpp

```diff
@@ -37,9 +38,9 @@ bool MessageLogWidget::userIsFemale() const
 void MessageLogWidget::logGameJoined(int gameId)
 {
     if (userIsFemale())
-        appendHtmlServerMessage(tr("You have joined game #%1.", "female").arg(gameId));
+        appendHtmlServerMessage(tr("You have joined game #%1.", "female").arg("<font color=\"blue\">"+ QString::number(gameId) + "</font>"));
```

Comment:
Oh dear god every line of this file is going to merge conflict in the gender branch

:finnadie: 

(Nothing to be done about it, carry on)